src/mpi/request/test.c

```diff
@@ -26,6 +26,39 @@ int MPI_Test(MPI_Request * request, int *flag, MPI_Status * status)
 #undef MPI_Test
 #define MPI_Test PMPI_Test
 
+#undef FUNCNAME
+#define FUNCNAME MPIR_Test_impl
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static inline int MPIR_Test_impl(MPIR_Request *request_ptr, int *flag, MPI_Status *status)
```

Comment:
No other is using this right ? If they ever use they should call MPIR_Test

Reply:
`MPID_Test` is using it.  The only reason it's not showing up is because you implemented it as a macro.  If you had used a `static inline` function, the compiler would have complained.